### ISSUE TITLE
Need Upgrades

### ISSUE THREAD

**Issue by Rahul-Rasve** [arafaysaleem/message_app#6]
**Describe the bug**
The app is using too old dependencies which do not support the current the SDK version and should be upgraded to the latest version along with all the dependencies used.

**To Reproduce**
1. Go to 'file/message_app'
2. In cmd, run 'flutter pub get'
3. See error

**Expected behavior**
Sdk version should be " >=2.12.4 <^3.1.0 "
fluashbar and flutter_link_preview is depricated
All the declared dependencies in pubspec.yaml do not support null safety.

**Comment by TechSmith90210** [comment]
Have you started working on this issue? @Rahul-Rasve

**Comment by Rahul-Rasve** [comment]
> Have you started working on this issue? @Rahul-Rasve

No, I want to contribute to the app, but it's too many errors right now.

**Comment by Priyank295** [comment]
i can try to upgrade message_app

**Comment by Rahul-Rasve** [comment]
@Priyank295 I have described how and what needs to be updated. Ping me once you have solved the issues.

**Comment by Priyank295** [comment]
> @Priyank295 I have described how and what needs to be updated. Ping me once you have solved the issues.

Ok

**Comment by noob-Coder-Harsh** [comment]
@arafaysaleem @Rahul-Rasve @Priyank295 guys let me know if you guys willing to fix this, i am also want to contribute in this

**Comment by bharathkumar-creator** [comment]
Hi there, guys I too want to contribute

**Comment by Shamteshi** [comment]
I've fixed all the errors concerning null safety issue and deprecated dependencies. However, I cannot run the app because googleservice.json file is missing.

**Comment by Shamteshi** [comment]
Nevermind, the app is running now. Had to create a new googlservice.json file, which I'll leave out of my pr so it doesn't conflict with yours.
